src/bundle/Resources/views/admin/content_type/modal_add_translation.html.twig

```diff
@@ -0,0 +1,32 @@
+{% trans_default_domain 'locationview' %}
+
+{% form_theme form '@ezdesign/form_fields.html.twig' '@ezdesign/content/translation_add_form_fields.html.twig' %}
+
+<div class="modal fade ez-modal ez-translation" id="add-translation-modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        {{ form_start(form, {'action': path('ezplatform.content_type.add_translation')}) }}
```

Comment:
This twig is the same as in the content translation.  The only one change is the acton. 
This should share the same twig and be passed one option `action`